### PR TITLE
fix: fix update to next version

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Instead of `SNAPSHOT`, use `next` prefix, to signify development version in `main` branch
Fix updating the next version, as PR may not get greated due to a specific error message

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/21044

### CODE REVIEW

File: packages/common/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eclipse-che/common",
-  "version": "7.37.0-SNAPSHOT",
+  "version": "7.43.0-SNAPSHOT",
```
Comment (akurinnoy) [inline packages/common/package.json:3]: shouldn't be this version changed to `x.y.z-next` as well?

**Review by nickboldt — APPROVED** [review]
not sure we need the -next suffix... but otherwise I like this.

For the .x branches, will they also have -next?

**Comment by mkuznyetsov** [comment]
@nickboldt next suffix could help to differentiate between tags and non-tags ...if that's needed ofc

**Comment by mkuznyetsov** [comment]
@nickboldt -next suffix in bugfix branch may simply provide a hint, that this code (or in the resulting artifact image) was built from the branch, and not tag. And that the next tag made from this branch would be the current version, just without suffix
if that makes sense

**Pull request merged**